compose.yaml

```diff
@@ -0,0 +1,27 @@
+name: "dradis-ce-production"
+
+services:
+  web:
+    # image: dradis/dradis-ce:latest
+    build:
+      context: .
+      platforms:
+        - linux/amd64
+    volumes:
+      # Persist the database
+      - dradis-ce-storage:/app/storage
+    ports:
+      - 443:443
+      - 80:80
+    environment:
+      # Set domain from ENV var so thruster can handle let's encrypt certificates
+      - TLS_DOMAIN=${TLS_DOMAIN}
```

Comment:
is this syntax really needed? wouldn't Docker do this by default?

Reply:
You're right it will automatically load all env vars in `.env` without the need for explicitly defining them here. Removed